pkg/install/v1alpha1/cstor_pool_0.7.0.go

```diff
@@ -386,25 +461,25 @@ metadata:
   name: cstor-pool-delete-liststoragepoolcr-default-0.7.0
 spec:
   meta: |
-    id: deletelistsp
+    id: liststoragepoolcr
     apiVersion: openebs.io/v1alpha1
     kind: StoragePool
     action: list
     options: |-
       labelSelector: openebs.io/storage-pool-claim={{.Storagepool.owner}}
   post: |
-    {{- $sps := jsonpath .JsonResult "{range .items[*]}pkey=sps,{@.metadata.name}=;{end}" | trim | default "" | splitList ";" -}}
-    {{- $sps | notFoundErr "storge pool cr not found" | saveIf "deletelistcsp.notFoundErr" .TaskResult | noop -}}
+    {{- $sps := jsonpath .JsonResult "{range .items[*]}pkey=sps,{@.metadata.name}="";{end}" | trim | default "" | splitList ";" -}}
```

Comment:
This will fail, I guess.
`=""`
Make it to `=` only.
The intention of `""` is to push empty. If you dont put `""` it will still put empty

Reply:
Please check for other such cases..

The only changes required to pull pre release yaml here are following to make runtasks work
1. Replace ` ` with " "
2. Escape ```. ``` charcters (e.g. in label keys ) by putting \\ befor the ```. ```character
3. The one specified above.

Reply:
Thank you! addressed.